doc/docs/security/selinux.md

```diff
@@ -0,0 +1,30 @@
+<!-- markdownlint-disable-file MD013-->
+# BlueChi's SELinux policy
+
+BlueChi provides a custom SELinux policy, limiting access of the `bluechi-controller` and `bluechi-agent`. It can be installed via
+
+```bash
+dnf install bluechi-selinux
+```
+
+By default, the package allows `bluechi-controller` to bind and `bluechi-agent` to connect to any port so that changes of the [ControllerPort](../man/bluechi-agent-conf.md#controllerport-uint16_t) are not being blocked.
+
+In order to allow BlueChi to only use one port, the usage of any needs to be disabled with [setsebool](https://linux.die.net/man/8/setsebool):
+
+```bash
+# Turn off the any policy for bluechi-controller
+setsebool -P bluechi_port_bind_any 0
+
+# Turn off the any policy for bluechi-agent
+setsebool -P bluechi_agent_port_connect_any 0
```

Comment:
Wouldn't be `bluechi_agent_connect_to_any_port` be a more descriptive name?

Reply:
I would keep the `_connect_any` part similar to the [systemd_socket_proxyd_connect_any](https://github.com/fedora-selinux/selinux-policy/blob/85bbb6d1bc69e84ea2440cff647e426b74f1b40a/policy/modules/system/systemd.te#L22C13-L22C46)

Reply:
OK